tests/JavascriptTests.cmake

```diff
@@ -181,6 +186,7 @@ function(add_web_client_test case specFile)
     "GIRDER_PORT=${web_client_port}"
     "MONGOD_EXECUTABLE=${MONGOD_EXECUTABLE}"
     "GIRDER_TEST_DATA_PREFIX=${GIRDER_EXTERNAL_DATA_ROOT}"
+    "${setup_database_env}"
```

Comment:
You mean `"GIRDER_TEST_DATABASE_CONFIG=${fn_SETUP_DATABASE}"` here?  The problem with that is the environment variable would be set.  I would have to modify the code [here](https://github.com/girder/girder/pull/2040/files/0f1fb6929af47fad4073a14d6c0f3d51e7778fed#diff-44b2005fe01184a1089900dedd126fd4R196) to detect if the var exists but is empty.  I'm willing to do so if you think it's better.